model_compression_toolkit/qunatizers_infrastructure/pytorch/quantize_wrapper.py

```diff
@@ -44,13 +43,20 @@ def __init__(self,
             self.dispatcher = dispatcher
             self.add_module('layer', module)
             self._weight_vars = []
+
+            # Init weights quantizers
             for name, quantizer in self.dispatcher.weight_quantizers.items():
                 weight = getattr(self.layer, name)
-                quantizer.initialize_quantization(weight.shape,
-                                                  name, self)
-
+                quantizer.initialize_quantization(weight.shape, name, self)
```

Comment:
why not save as a dictionary, as in keras?

Reply:
it's weight not activation (not this PR). We will talk on init in the meeting.